src/frontend/js/utils/useDateFormat.tsx

```diff
@@ -1,5 +1,11 @@
 import { type FormatDateOptions, useIntl } from 'react-intl';
 
+export const FORMAT_DATE_OPTIONS: FormatDateOptions = {
```

Comment:
What do you think ?
```suggestion
export const DEFAULT_DATE_FORMAT: FormatDateOptions = {
```

Reply:
Seems legit :) I love putting "type-hint" in names 🥸